src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -6174,15 +6174,15 @@ public object VisitMemberExpression(MemberExpressionAst memberExpressionAst)
             }
 
             var target = CompileExpressionOperand(memberExpressionAst.Expression);
+
+            // If the ?. operator is used for null conditional check, add the null conditional expression.
             var memberNameAst = memberExpressionAst.Member as StringConstantExpressionAst;
-            if (memberNameAst != null)
-            {
-                string name = memberNameAst.Value;
-                return DynamicExpression.Dynamic(PSGetMemberBinder.Get(name, _memberFunctionType, memberExpressionAst.Static), typeof(object), target);
-            }
+            Expression memberAccessExpr = memberNameAst != null
+                ? DynamicExpression.Dynamic(
+                    PSGetMemberBinder.Get(memberNameAst.Value, _memberFunctionType, memberExpressionAst.Static), typeof(object), target)
```

Comment:
It seems unnecessary to have this on a new line, given the `DynamicExpression.Dynamic` below has similar length.

Reply:
Fixed.